### ISSUE TITLE
Questions on training on my own dataset

### ISSUE THREAD

**Issue by Noone65536** [HRNet/HRNet-Object-Detection#8]
I tried to use it to train on my own  VOC dataset, which has 10 classes. I modified the config files and the voc.py. However, the results was very bad.  The loss decreased but stopped at around 0.5. And also, I can not use your pre-train model because the class number is different. 

How could I modify the per-train model ? If it's no able to do that, did I do something wrong when use the hrnet on my own dataset? I did the followings: 
1) made the config file suitable for the VOC format dataset (as in pascal_voc)
2)changed the class number in config file
3)changed the classes' name in voc.py

thank you for reading my question. I'm new in CV and please forgive me if the quetion is  stupid

**Comment by wondervictor** [comment]
Have you checked your input data format and the annotation format?

**Comment by wondervictor** [comment]
closed for no more response
